### PR TITLE
Added YouTube search

### PR DESCRIPTION
**Pull request by Krishna-Singhal**

### CODE REVIEW

File: userge/plugins/utils/voice_call.py [hunk 1]
```diff
@@ -172,12 +174,28 @@ async def play_music(msg: Message):
...
             )
-            await reply_text(
-                msg,
-                text
-            )
+            await reply_text(msg, text)
         else:
-            await reply_text(msg, "Only youtube links are supported")
+            mesg = await reply_text(msg, f"Searching `{msg.input_str}` on YouTube")
+            results = VideosSearch(msg.input_str, limit=1).result()['result']
```
Comment (rking32) [inline userge/plugins/utils/voice_call.py:180]: this is a sync call

Comment (Krishna-Singhal) [reply userge/plugins/utils/voice_call.py:180]: 🤔🤔

File: userge/plugins/utils/voice_call.py [hunk 2]
```diff
@@ -383,6 +397,9 @@ async def yt_down(msg: Message):
     )
     shutil.rmtree("temp_music_dir", ignore_errors=True)
 
+    if (await msg.client.get_me()).id == msg.from_user.id:
```
Comment (rking32) [inline userge/plugins/utils/voice_call.py:400]:
```suggestion
    if msg.client.id == msg.from_user.id:
```
why extra request ?

Comment (Harsh-br0) [reply userge/plugins/utils/voice_call.py]: Bruh 🤔

Comment (Harsh-br0) [reply userge/plugins/utils/voice_call.py]: Client object doesn't have any id attr ig

**Pull request merged**